api/api.proto

```diff
@@ -2597,7 +2814,7 @@ message SdkVersion {
     // SDK version minor value of this specification
     Minor = 34;
```

Comment:
This seems to be incorrect. According to the SDK CHANGELOG, this is supposed to be `33`, but I see `34`. It is supposed to be 0.33.1, right?